### DISCUSSION TITLE
How to fetch email address of all my group members?

### DISCUSSION THREAD

**Discussion by seetharamanjkpt** [community/community#24527] [Apps, API and Webhooks]
<p>Dear Team,</p>
<p>I tried with /users/&lt;userid&gt;, /emails, /public_emails/ and few other API end urls, but nothing fruitful.  Even after all my group members added their public email, these API calls returns their email address as empty, null.  Is that allowed/possible to retrive all my group members mail ids with any API ? or the APIs are just to get/set only my profile details ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f5c7b5b0523405b4c5cda8dac719aaf0da3435f4.jpeg" data-download-href="/uploads/short-url/z4gR5RR1LtA8Cg38okmpGNyecOo.jpg?dl=1" title="profile.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/64897987/181085180-d7a16357-eabe-48a1-a54a-fa7c9d9b1699.jpg" alt="profile.jpg" data-base62-sha1="z4gR5RR1LtA8Cg38okmpGNyecOo" width="690" height="472" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f5c7b5b0523405b4c5cda8dac719aaf0da3435f4_2_690x472.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/f/f5c7b5b0523405b4c5cda8dac719aaf0da3435f4_2_1035x708.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f5c7b5b0523405b4c5cda8dac719aaf0da3435f4.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f5c7b5b0523405b4c5cda8dac719aaf0da3435f4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">profile.jpg</span><span class="informations">1198×820 101 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by byrneh** [comment, marked as answer]
<p>Hi,</p>
<p>I am able to retrieve your email address using REST API<br>
<a href="https://api.github.com/users/seetharamanjkpt" class="onebox" target="_blank" rel="nofollow noopener">https://api.github.com/users/seetharamanjkpt</a></p>
<p>WARNING: You only see publicly visible email addresses when authenticated with GitHub.<br>
So, if if call the above URL unauthenticated, email address is null<br>
Performing the same authenticated (passing access token for example), the email is shown<br>
For more information, on Authentication (<a href="https://developer.github.com/v3/#authentication" rel="nofollow noopener">https://developer.github.com/v3/#authentication</a>).</p>
